submissions/Oleksii_Drohachov/DOM_task/script.js

```diff
@@ -0,0 +1,164 @@
+    altName: "Lord of Excess",
+    warCry:
+      '"Embrace your hunger, your lust, your desire. The universe is ours for the taking!"',
+    descr:
+      '&emsp;Slaanesh, also known as the "Dark Prince," the "Prince of Pleasure," the "Lord of Excess," the "Perfect Prince," and "Prince of Chaos" in the Imperium of Man and "She Who Thirsts" among the Aeldari, is the Chaos God of pleasure, hedonism, excess and decadence. Lust, pride and self-indulgence are the hallmarks of all who follow it. <br><br>&emsp; Slaanesh is the Lord of Pleasure, the Dark God dedicated to the pursuit of earthly gratification and the overthrow of all decent behaviour, as well as hedonism and pleasure for its own sake. It is the god of obsession, the Master of Excess in All Things, from gluttony to lust to megalomania. Its sacred number is six and the colours associated with Slaanesh are riotous purples, pinks and black. The daemonic armies of Slaanesh are known as the Legions of Excess.',
+    mainLegion: "Main legion: Emperors children",
+  },
+];
+
+function makeFirstPage() {
+  const container = document.createElement("div");
+  container.classList.add("first__page");
+
+  const firstImg = document.createElement("img");
+  firstImg.setAttribute("srcset", greetings.imgUrl);
+  firstImg.classList.add("content__image");
+
+  const text = document.createElement("p");
+  text.classList.add("description");
+  text.innerHTML = greetings.text;
+
+  container.append(firstImg);
+  container.append(text);
+  contentBox.innerHTML = "";
+  contentBox.append(container);
+}
+
+makeFirstPage();
+
+function makeMenu(content) {
+  for (let i = 0; i < content.length; i++) {
+    const item = document.createElement("li");
+    const menuLink = document.createElement("a");
+    item.classList.add("menu__item");
+    item.setAttribute("id", `${i}`);
+    menuLink.setAttribute("id", `${i}`);
+    menuLink.setAttribute("href", "#");
+    menuLink.classList.add("menu__link");
+    menuLink.textContent = content[i].name;
+    item.append(menuLink);
+    menuList.append(item);
+  }
+
+  menuWrapper.append(menuList);
+}
+
+makeMenu(content);
+
+burger.addEventListener("click", menuHandler);
+const menuClasses = menuList.classList;
+
+function menuHandler(e) {
+  function openMenu() {
+    menuClasses.remove("menu__list_hidden");
+  }
+
+  function closeMenu() {
+    menuClasses.add("menu__list_hidden");
+  }
+
+  function closeMenuByTap(e) {
+    if (!e.target.classList.contains("menu__item")) {
+      menuClasses.add("menu__list_hidden");
+    }
+  }
+
+  if (menuClasses.contains("menu__list_hidden")) {
+    openMenu();
+    contentBox.addEventListener("click", closeMenuByTap);
+  } else {
+    closeMenu();
+    contentBox.removeEventListener("click", closeMenuByTap);
+  }
+}
+
+const menuItems = document.querySelectorAll(".menu__item");
+
+for (let i = 0; i < content.length; i++) {
+  menuItems[i].addEventListener("click", makeContent);
+}
+
+function makeContent(e) {
+  menuHandler();
```

Comment:
I am confused, are you redoing your function for the menu every time you go to another page? Isn't `menuHandler` supposed to receive the `e` parameter (tip: it does but it shouldn't)? Why not run this once somewhere?